src/test/java/it/pagopa/pn/bff/pnclient/externalregistries/PnExternalRegistriesClientImpTest.java

```diff
@@ -0,0 +1,118 @@
+package it.pagopa.pn.bff.pnclient.externalregistries;
```

Comment:
Rename it as PnExternalRegistriesClientImplTest and add the PnExternalRegistriesClientImplTestIT as done for the others